### PR TITLE
Reorganize Example ML Apps Documentation

### PR DESCRIPTION
**Pull request by tylerhutcherson**
- Extract important concpets higher in the chain.
- Add more links and images.
- Clean up wording and reduce volume of words overall.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,80 @@
 # Skafos Example ML Apps
+<img src="https://skafos.ai/wp-content/uploads/2019/05/skafos_horizontal_on_white_beta@1x.svg" width="100%">
```
Comment (eecaro) [inline README.md]: Do we have a smaller logo? If not, I would suggest putting it above the `# Skafos Example ML Apps`

File: README.md [hunk 2]
```diff
@@ -1,27 +1,80 @@
...
+*We are committed to building a platform that integrates with popular Machine Learning (ML) training frameworks & on-device ML formats, to create the best possible user experience delivering models to the edge.*
 
-Unfortunately, because of all these disparate pieces, it can sometimes be challenging to put together an end-to-end machine learning architecture for a specific use case. At Skafos, we are here to help with this process.
+With the wide array of tools and technologies available, **it is challenging to develop an end-to-end machine learning architecture for mobile ML deployment**.
 
-In this repository, we are building out example model training workflows and app integrations for different frameworks. Each architecture is paired with an app that leverages the Skafos framework to deliver new models to these apps. This repo is currently under active development, so check back often if you don't see your favorite framework or use case. It is likely to be here soon!
+## Overview
+In this repository, we're assembling example workflows that demonstrate all parts of the end-to-end pipeline: *model training*, *delivery*, and *mobile app integration*.
```
Comment (eecaro) [inline README.md]: Does model training include data collection and formatting or is that something we don't include/isn't necessary at the moment?

File: TensorFlow/tflite/ios/README.md [hunk 3]
```diff
@@ -1,30 +1,56 @@
...
+### 2. iOS App
+We've provided a pre-packed Xcode project for you to work with. All project contents can be found in the `app/` subdirectory. *The app is based on [TensorFlow's example](https://github.com/tensorflow/examples/tree/master/lite/examples/image_classification/ios), but we've integrated the [Skafos iOS Framework](https://github.com/skafos/ios) to handle model updates over-the-air.*
+
+Make sure you have the following requirements satisfied:
+
+- Device with iOS 12.0 or above
+- Xcode 10.0 or above
+- Valid Apple Developer ID
+- Xcode command-line tools (run xcode-select --install)
+- CocoaPods (run bash sudo gem install cocoapods)
```
Comment (eecaro) [inline TensorFlow/tflite/ios/README.md]: should the `xcode-select` and `gem install` commands be in code text?

File: Keras/coreml/ios/README.md [hunk 4]
```diff
@@ -1,31 +1,58 @@
...
+### 2. iOS App
+We've provided a pre-packed Xcode project for you to work with. All project contents can be found in the `app/` subdirectory.
+
+Make sure you have the following requirements satisfied:
+
+- Device with iOS 12.0 or above
+- Xcode 10.0 or above
+- Valid Apple Developer ID
+- Xcode command-line tools (run xcode-select --install)
+- CocoaPods (run bash sudo gem install cocoapods)
```
Comment (eecaro) [inline Keras/coreml/ios/README.md]: should the `xcode-select` and `gem install` commands be in code text?

File: README.md (hunk 2 above)
Comment (tylerhutcherson) [reply README.md]: certainly an important part, but not exactly the focus of these examples

File: TensorFlow/tflite/ios/README.md (hunk 3 above)
Comment (tylerhutcherson) [reply TensorFlow/tflite/ios/README.md]: yep!

File: Keras/coreml/ios/README.md (hunk 4 above)
Comment (tylerhutcherson) [reply Keras/coreml/ios/README.md]: yep!

**Pull request merged**